levelZeroLib/src/levelZeroContext.cpp

```diff
@@ -98,7 +106,11 @@ JNIEXPORT jint JNICALL Java_uk_ac_manchester_tornado_drivers_spirv_levelzero_Lev
     ze_command_queue_desc_t cmdQueueDesc = {};
     jclass commandDescriptorClass = env->GetObjectClass(javaCommandQueueDescriptor);
     field = env->GetFieldID(commandDescriptorClass, "ptrZeCommandDescriptor", "J");
+#ifdef _WIN32
+    int64_t ptrZeCommandDescriptor = env->GetLongField(javaCommandQueueDescriptor, field);
+#else
     long ptrZeCommandDescriptor = env->GetLongField(javaCommandQueueDescriptor, field);
```

Comment:
I think we should also use `int64_t` in Linux as well. Let's keep it for now, I will make a note and do this after we merge, so we can unify the Windows and Linux ports as much as possible.